### DISCUSSION TITLE
Does this gem support chartjs?

### DISCUSSION THREAD

**Discussion by NyazJaff** [prawnpdf/prawn#1329] [General] [answered]
Hi all

I'm after a gem to generate pdfs and was wondering if this gem supports Chartjs4 at all? 

Thanks

**Comment by pointlessone** [comment, marked as answer]
No.

**Reply by NyazJaff** [reply]
okay thanks,

How about rendering Base64Image please?

**Reply by pointlessone** [reply]
What is Base64Image? Prawn supports PNG and JPEG images.
